File: eBotanika.Mobile/lib/models/biljke.dart
class Biljke {
  int biljkaID;
  int kategorijaID;
  String naziv;
  String opis;
  String slika;
  String cijena;

  Biljke(
      {required this.biljkaID,
      required this.kategorijaID,
      required this.naziv,
      required this.opis,
      required this.slika,
      required this.cijena});

  factory Biljke.fromJson(Map<String, dynamic> json) {
    return Biljke(
      biljkaID: int.parse(json["biljkaID"].toString()),
      kategorijaID: int.parse(json["kategorijaID"].toString()),
      naziv: json["naziv"],
      opis: json["opis"],
      slika: json["slika"],
      cijena: json["cijena"],
    );
  }

  Map<String, dynamic> toJson() => {
        "biljkaID": biljkaID,
        "kategorijaID": kategorijaID,
        "naziv": naziv,
        "opis": opis,
        "slika": slika,
        "cijena": cijena
      };
}

File: eBotanika.Services/Database/Biljke.cs
namespace eBotanika.Services.Database
{
    public partial class Biljke
    {
        public Biljke()
        {
            Ocjena = new HashSet<Ocjena>();
            Rezervacije = new HashSet<Rezervacije>();
        }

        public int BiljkeID { get; set; }
        public int KategorijaID { get; set; }
        public Kategorija Kategorija { get; set; }
        public int UposlenikID { get; set; }
        public Uposlenik Uposlenik { get; set; }
        public string Naziv { get; set; }
        public string Opis { get; set; }
        public string? Slika { get; set; }
        public decimal Cijena { get; set; }
        public ICollection<Ocjena> Ocjena { get; set; }
        public ICollection<Rezervacije> Rezervacije { get; set; }
    }
}
